src/Microsoft.Health.Fhir.SqlServer/Resources.resx

```diff
@@ -126,4 +126,10 @@
   <data name="CyclicIncludeIterateNotSupported" xml:space="preserve">
     <value>Cyclic include iterate queries are not supported.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SchemaVersionNeedsToBeUpgraded" xml:space="preserve">
+    <value>Search parameter status information should not be null.</value>
```

Comment:
>Search parameter status information should not be null [](start = 11, length = 54)

Name and content is sound misalign for me.
How status is null for search param and Schema needs to be upgraded related to eachover?

Reply:
Thanks for catching this. Fixed in 5a32607.